supervisor/backups/backup.py

```diff
@@ -677,12 +707,12 @@ async def store_homeassistant(self, exclude_database: bool = False):
             ATTR_EXCLUDE_DATABASE: exclude_database,
         }
 
+        tar_name = f"homeassistant.tar{'.gz' if self.compressed else ''}"
         # Backup Home Assistant Core config directory
-        tar_name = Path(
-            self._tmp.name, f"homeassistant.tar{'.gz' if self.compressed else ''}"
-        )
-        homeassistant_file = SecureTarFile(
-            tar_name, "w", key=self._key, gzip=self.compressed, bufsize=BUF_SIZE
```

Comment:
bufsize is now set in the outer tar and the inner tar uses the bufsize of the outer tar